portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DefinitionsSearch.testcase

```diff
@@ -23,6 +23,15 @@ definition {
 
 		ACDefinitions.deleteAllSearchQuery();
 
+		ACUtils.tearDownDXP();
```

Comment:
Hey @joshitagaki, it's not possible to use this `tearDown` macro yet. I'm working on this [ticket](https://liferay.atlassian.net/browse/LRAC-13440) to rehabilitate her. Could you change to [this](https://github.com/liferay-ac/liferay-portal/pull/735/files#diff-83fef259c7778902d9e334672bbfb1ebfd03d8c121bd5fb87aa09c64193de939R35-R37)?

Reply:
I'm not sure I understand. This teardown macro seems to be working fine. It's only the other teardown for AC right?

Reply:
I think we disabled all uses of this macro since it was not possible to delete property and data source and this was causing errors in the tests since the delete button did not appear. We did that [here](https://github.com/liferay-ac/liferay-portal/pull/370).

So if you enable this macro, your automation will fail since in the upstream suite it will only be possible to delete the property and not the data source. I'm working on this [ticket](https://liferay.atlassian.net/browse/LRAC-13440) to enable this macro again just by deleting the properties created since we enabled this functionality again in this [story](https://liferay.atlassian.net/browse/LRAC-14082).

So I think you should change this part of the code to look like this [here](https://github.com/liferay-ac/liferay-portal/pull/735/files#diff-83fef259c7778902d9e334672bbfb1ebfd03d8c121bd5fb87aa09c64193de939L26-L28)
```suggestion
		// It is not possible to delete properties and data source after LRAC-13389. This functionality should return after LRAC-8040

		//ACUtils.tearDownAC();
```

Reply:
Ahh yes but the deletion of property and data source is part of the `ACUtils.tearDownAC();` and not the `ACUtils.tearDownDXP();` right? Since `ACUtils.tearDownDXP();` only disconnects AC from DXP. Or maybe I'm missing something from this comment lol. Let's get on a call to discuss.

Reply:
Ohhhhh man, I'm sorry . I was reading `tearDownAC` in this part and not `tearDownDXP` hahahahaha. You're right, I was looking wrong 😆 🤣